apps/etterlatte-saksbehandling-ui/client/src/components/person/journalfoeringsoppgave/kontroll/KontrollerOppgave.tsx

```diff
@@ -149,14 +174,16 @@ export default function KontrollerOppgave() {
                   )
                 }
               />
-            </div>
+            </>
           )}
         </>
       )}
 
+      {error && <ErrorMessage>{error}</ErrorMessage>}
+
       <div>
         <FlexRow justify="center" $spacing>
-          <Button variant="primary" onClick={neste} disabled={!kanGaaVidere}>
+          <Button variant="primary" onClick={neste} disabled={samsvar === JaNei.NEI}>
```

Comment:
Trenger vi disabled? Når `neste()` setter feilmeldinger så får man ikke se de hvis man ikke får trykke på knappen :p Ser ut som `samsvar !== JaNei.JA` er en del av den sjekken allerede

Reply:
Hvis SB velger NEI får de opp en stor rød alert. Så det er bare for å indikere at de da ikke kan gjøre noe som helst. Ingen vits å la de trykke knappen når hele flyten stopper pga. manglende samsvar

Reply:
![Screenshot 2023-09-27 at 09 59 07](https://github.com/navikt/pensjon-etterlatte-saksbehandling/assets/1224956/2b63c65b-6179-4a24-95b1-141aa0ed2593)

Reply:
Aight, da gir det mening!